routes/projects.tsx

```diff
@@ -9,38 +11,71 @@ const DEVICT_REPOS: string[] = [
   "devict/help",
 ];
 
-// const COMMUNITY_REPOS: string[] = [];
-// const REPOS = [...DEVICT_REPOS, ...COMMUNITY_REPOS];
+const COMMUNITY_REPOS: string[] = [
+  "benblankley/fort-rpg",
+  "blunket/camelot",
+  "imacrayon/alpine-ajax",
+  "imacrayon/snowbodyknows",
+  "imacrayon/whatthetofu",
+  "kentonh/ProjectNameGenerator",
+  "sethetter/linktrap",
+  "sethetter/reqq"
+];
 
 export default async function Home() {
+  const ALL_REPOS = DEVICT_REPOS.concat(COMMUNITY_REPOS);
   const ghToken = Deno.env.get("GITHUB_TOKEN");
   assert(ghToken);
 
   const issues = await getHelpWantedIssues({
     token: ghToken,
-    repos: DEVICT_REPOS,
+    repos: ALL_REPOS,
   });
 
+  const issuesListProps = {
+    repos: ALL_REPOS,
+    issues
+  };
+
+  const shuffle = (array: string[]) => { 
+    for (let i = array.length - 1; i > 0; i--) { 
+      const j = Math.floor(Math.random() * (i + 1)); 
+      [array[i], array[j]] = [array[j], array[i]]; 
+    } 
+    return array; 
+  }; 
+
   return (
-    <>
+    <div class="container mx-auto px-4">
       <h1 class="text-4xl font-bold mb-4">Contribute to devICT</h1>
-      <ul class="list-none">
-        {issues.map((issue) => {
-          const [orgName, repoName] = issue.repository_url.split("/").slice(-2);
-          const repoPath = `${orgName}/${repoName}`;
-          const repoUrl = `https://github.com/${repoPath}`;
-          return (
-            <li class="my-1">
-              <span class="font-bold underline hover:text-gray-600">
-                <a href={repoUrl}>{repoPath}</a>:{" "}
-              </span>
-              <span>
-                <a href={issue.html_url} class="underline text-blue-600 hover:text-blue-800 visited:text-purple-600">{issue.title}</a>
-              </span>
-            </li>
-          );
-        })}
-      </ul>
-    </>
+      <div class="grid grid-cols-4 gap-4">
+        <div class="col-span-3">
+          <IssuesList {...issuesListProps} />
+        </div>
+        <div>
+          <div class="mb-4">
+            <Card title="DevICT Repos">
+              <ul class="style-none">
+                {DEVICT_REPOS.map(repo => {
+                  const repoName = repo.split("/").slice(-1);
+                  const repoUrl = `https://github.com/${repo}`;
+                  return <li class="hover:underline"><a href={repoUrl} target="_blank">{repoName}</a></li>;
+                })}
```

Comment:
This looks ripe to be made it's own small component and reused for both devICT and community repos 😄